### ISSUE TITLE
Extra ORDER BY added on updateById call

### ISSUE THREAD

**Issue by erkiesken** [typeorm/typeorm#1804]
**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[x] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `websql`

**TypeORM version:**

[ ] `latest`
[ ] `@next`
[x] `0.1.16` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

When `@Entity` decorator has `orderBy` option set, then calling `updateById` on this entity will erroneously add `ORDER BY` to the `UPDATE` statement.

Here's a full snippet of code that triggers it:

```typescript
import {
  Entity, PrimaryColumn, Column,
  createConnection, Connection,
} from 'typeorm';

@Entity({
  name: 'test_entity',
  orderBy: {
    sequence_number: 'ASC',
  },
})
class TestEntity {
  @PrimaryColumn('integer')
  id: number;

  @Column('integer')
  sequence_number: number;
}

async function runTest() {
  const conn = await createConnection({
    type: 'postgres',
    logging: true,
    username: 'postgres',
    database: 'typeorm_testing',
    synchronize: true,
    entities: [TestEntity],
  });
  const repo = conn.getRepository(TestEntity);

  console.log('updating entity:');
  await repo.updateById(1, { sequence_number: 0 });
  console.log('entity updated');
};

if (require.main === module) {
  runTest().catch((err) => {
    console.error(err);
    process.exit(1);
  });
}
```

And in output logs:
```executing query:  UPDATE "test_entity" SET "sequence_number" = $1 WHERE "id"=$2 ORDER BY test_entity.sequence_number ASC -- PARAMETERS: [0,1]
query failed:  UPDATE "test_entity" SET "sequence_number" = $1 WHERE "id"=$2 ORDER BY test_entity.sequence_number ASC -- PARAMETERS: [0,1]
error:  { error: syntax error at or near "ORDER"
    at Connection.parseE (/Users/erki/Workspace/typeorm-issues/node_modules/pg/lib/connection.js:545:11)
    at Connection.parseMessage (/Users/erki/Workspace/typeorm-issues/node_modules/pg/lib/connection.js:370:19)
    at Socket.<anonymous> (/Users/erki/Workspace/typeorm-issues/node_modules/pg/lib/connection.js:113:22)
    at Socket.emit (events.js:180:13)
    at Socket.emit (domain.js:421:20)
    at addChunk (_stream_readable.js:269:12)
    at readableAddChunk (_stream_readable.js:256:11)
    at Socket.Readable.push (_stream_readable.js:213:10)
    at TCP.onread (net.js:578:20)
  name: 'error',
  length: 94,
  severity: 'ERROR',
  code: '42601',
  detail: undefined,
  hint: undefined,
  position: '63',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'scan.l',
  line: '1087',
  routine: 'scanner_yyerror' }
{ QueryFailedError: syntax error at or near "ORDER"
    at new QueryFailedError (/Users/erki/Workspace/typeorm-issues/src/error/QueryFailedError.ts:7:9)
    at Query.callback (/Users/erki/Workspace/typeorm-issues/src/driver/postgres/PostgresQueryRunner.ts:216:26)
    at Query.handleError (/Users/erki/Workspace/typeorm-issues/node_modules/pg/lib/query.js:143:17)
    at Connection.connectedErrorHandler (/Users/erki/Workspace/typeorm-issues/node_modules/pg/lib/client.js:132:26)
    at Connection.emit (events.js:180:13)
    at Connection.emit (domain.js:421:20)
    at Socket.<anonymous> (/Users/erki/Workspace/typeorm-issues/node_modules/pg/lib/connection.js:117:12)
    at Socket.emit (events.js:180:13)
    at Socket.emit (domain.js:421:20)
    at addChunk (_stream_readable.js:269:12)
  message: 'syntax error at or near "ORDER"',
  name: 'QueryFailedError',
  length: 94,
  severity: 'ERROR',
  code: '42601',
  detail: undefined,
  hint: undefined,
  position: '63',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'scan.l',
  line: '1087',
  routine: 'scanner_yyerror',
  query: 'UPDATE "test_entity" SET "sequence_number" = $1 WHERE "id"=$2 ORDER BY test_entity.sequence_number ASC',
  parameters: [ 0, 1 ] }```

**Comment by erkiesken** [comment]
Documentation says the `orderBy` should be only used for `find` operations:

http://typeorm.io/#/decorator-reference/entity

> orderBy - specifies default ordering for entities when using find operations and QueryBuilder.

**Comment by pleerock** [comment]
It was a bug, fixed in `0.1.17`.
